src/containers/documentation/ApiOverviewPage.tsx

```diff
@@ -3,13 +3,38 @@ import * as React from 'react';
 import { Helmet } from 'react-helmet';
 import { Link, useParams } from 'react-router-dom';
 import ReactMarkdown from 'react-markdown';
+
 import { PageHeader } from '../../components';
+import { APIDescription } from '../../apiDefs/schema';
 import { APIUrlSlug } from '../../types';
+import { lookupCategoryByApi } from '../../apiDefs/query';
```

Comment:
Replace `lookupCategoryByApi` with `lookupApiCategory`.
```suggestion
import { lookupApiCategory } from '../../apiDefs/query';
```